examples/event_handling/coords_demo.py

```diff
@@ -29,7 +29,6 @@ def on_move(event):
     if event.inaxes:
         # get the x and y pixel coords
         x, y = event.x, event.y
-        ax = event.inaxes  # the axes instance
         print('data coords %f %f, pixel coords %f %f'
                 % (event.xdata, event.ydata, x, y))
```

Comment:
you could just inline event.x, event.y here.

Reply:
(and f-stringify)